Modules/AuctionManager/AuctionManager.lua

```diff
@@ -218,6 +235,9 @@ function AuctionManager:StartAuction(itemId, itemLink, itemSlot, baseValue, maxV
         auctionMessage = auctionMessage .. string.format(CLM.L["Anti-snipe time: %s."], tostring(self.antiSnipe))
     end
     SendChatMessage(auctionMessage , "RAID_WARNING")
+    if CLM.GlobalConfigs:GetAllowChatCommands() then
```

Comment:
Well I'm not sure If we should be doing this. Posting this message everytime auction starts suggests people should do `!bid`. And this is not the flow expected by addon  but a fallback way of doing stuff (which i don't like tbh, but i don't think there is any other way to do that than through chat).

Reply:
Ideally everyone would have the addon installed and I'd not need to have the option enabled.  I always seem to run into the issue where people don't know they can whisper me with a bid, and then they don't remember what the command is. My last commit adds a toggle for the message separate from whether or not the `!bid` command is enabled.